src/main/java/org/folio/service/EcsTlrService.java

```diff
@@ -8,4 +8,6 @@
 public interface EcsTlrService {
   Optional<EcsTlr> get(UUID requestId);
   EcsTlr post(EcsTlr ecsTlr);
+  boolean put(UUID requestId, EcsTlr ecsTlr);
+  boolean delete(UUID requestId);
```

Comment:
Service has no concept of POSTing and PUTing. Those are terms only make sense in context of controllers and clients. Service creates and updates objects instead. I would suggest renaming these two methods to `create` and `update` respectively.